### PR TITLE
Add writing guidelines

### PR DESCRIPTION
**Pull request by stickies-v**
Condensed my observations from the first cohort into these guidelines. Open for comments/suggestion!

### CODE REVIEW

File: writing-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+
+It's important to have a well-defined understanding of who you think will be the main audience that can benefit from your writing. Be explicit (at least with yourself, but it can help to define this in the article too) about which knowledge you expect them to have, what you expect them to learn, and what you think could be the best way to do so.
+
+If you define your audience too broadly, you will upset your actual audience as you would be when reading "The Bitcoin Script Expert". Conversely, if you define your audience too narrowly, you may lose some readers that otherwise would have benefited from your writing, but are lacking just a bit too much background knowledge to follow along.
+
+Defining your audience and finding a good balance between assuming not too much and not too little pre-existing knowledge is a delicate exercise, but one that needs to be done anyway.
+
+### 4.2. The Qala target audience
+
+The Qala program has a very clear technical focus. Most of your writing should reflect that. Even though articles catered for a general audience can be helpful (and are important for the Bitcoin ecosystem in general), these will often not be the best choice for two reasons:
```
Comment (adamjonas) [inline writing-guidelines.md:113]:
```suggestion
The Qala program has a very clear technical focus. Most of your writing should reflect that. Even though articles catered to a general audience can be helpful (and are important for the Bitcoin ecosystem in general), these will often not be the best choice for two reasons:
```

File: writing-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+> 
+> https://www.ox.ac.uk/students/academic/guidance/skills/plagiarism
+
+When presenting an idea that is not yours, you'll have to choose between using your own wording or quoting/paraphrasing the original author. You should always strive to rephrase a topic in your own words (e.g. by phrasing from it an alternative perspective, or in simpler terms, or by combining it with other topics). Sometimes rephrasing is not possible or sensible. In that case, it is okay to quote or paraphrase, but you *always* need to clearly indicate which content is quoted/paraphrased and reference the source.
+
+Even though plagiarism is often not illegal, it is almost guaranteed to blemish your reputation if and when it is discovered. Especially in an open-source community such as Bitcoin's, be generous with your references. It ensures the original author receives credit for their work where it is due. As an additional upside, ample linking to high-quality sources is a great way to make your own article more useful. Even if your writing is sufficiently original, it's usually a good idea to consistently reference the sources that inspired your thinking.
```
Comment (adamjonas) [inline writing-guidelines.md:146]: Maybe add that is shows a laziness because it takes credit for other's work without creating value.

**Review by adamjonas — COMMENTED** [review]
well done

File: writing-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+It's important to have a well-defined understanding of who you think will be the main audience that can benefit from your writing. Be explicit (at least with yourself, but it can help to define this in the article too) about which knowledge you expect them to have, what you expect them to learn, and what you think could be the best way to do so.
+
+If you define your audience too broadly, you will upset your actual audience as you would be when reading "The Bitcoin Script Expert". Conversely, if you define your audience too narrowly, you may lose some readers that otherwise would have benefited from your writing, but are lacking just a bit too much background knowledge to follow along.
+
+Defining your audience and finding a good balance between assuming not too much and not too little pre-existing knowledge is a delicate exercise, but one that needs to be done anyway.
```
[the remark this answers is not in the document]
Comment (stickies-v) [reply writing-guidelines.md:109]: Fixed in 0d06ec6, thanks!

File: writing-guidelines.md (hunk 2 above)
Comment (stickies-v) [reply writing-guidelines.md]: Fixed in 50ed939, thanks!

**Comment by stickies-v** [comment]
Thank you both for your review, I've incorporated all suggestions and also added some more of my own. I'll leave the PR up for another day or so and if no further comments I'll merge, we can always improve later.
